src/lib/pipeline/ExecuteUnit.cc

```diff
@@ -215,13 +204,6 @@ void ExecuteUnit::purgeFlushed() {
   }
 }
 
-uint64_t ExecuteUnit::getBranchExecutedCount() const {
-  return branchesExecuted_;
-}
-uint64_t ExecuteUnit::getBranchMispredictedCount() const {
-  return branchMispredicts_;
-}
-
```

Comment:
Why did we need to get rid of the branchesExecuted statistic? It isn't dependent on the execute unit "holding" the BP and all of the previous logic remains in place from before other than updating the counter

Reply:
PMU events don't track branches executed (at least according to the description of PMU events documented by fujitsu for the a64fx), just the number of branches and the number of mispredictions.  Therefore, this has been replaced with a branchesFetched statistic (recorded by the fetch unit) to better align with real hardware metrics.